renormalizer/utils/basis.py

```diff
@@ -37,11 +42,23 @@ class BasisSHO(BasisSet):
         x0 (float): the origin of the harmonic oscillator. Default = 0.
     """
     
-    def __init__(self, omega, nbas, x0=0.):
+    def __init__(self, omega, nbas, x0=0., dvr=False):
         self.omega = omega
         self.x0 = x0  # origin = x0
         super().__init__(nbas, [0] * nbas)
         
+        # calculate x, p power generally, but not efficient for x/p or x^2/p^2
+        # which have been hard-coded already.
+        self.general_xp_power = False
```

Comment:
Why not add an option for this? This way seems to be error-prone as demonstrated in the bug fix commit 25decac.
Also, the document is not updated.
I can probably help on these things though.

Reply:
Yes. You are right. I will fix this and update the document.